com.jianguoyun.Nutstore.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Encoding=UTF-8
+Type=Application
+Terminal=false
+Icon=com.jianguoyun.Nutstore
+Exec=nutstore
+Name=Nutstore
+Name[zh_CN]=坚果云 
+Comment=Data Sync, Sharing, Backup
+Comment[zh_CN]=数据同步,共享和备份
+Categories=Network;Application;
```

Comment:
I don't think there's an Application category, see here for the list of possible values https://specifications.freedesktop.org/menu-spec/latest/apa.html